example/text-links/index.html

```diff
@@ -26,14 +26,6 @@
           // ignore unbound links
           if (typeof start !== 'object' || typeof end !== 'object') return;
 
-          // draw link line
```

Comment:
For the links to be drawn I think it needs the prop: `.linkCanvasObjectMode(() => 'after')`.
Also the `globalScale` is no longer used and can be removed from the args.